src/tests/Equinor.ProCoSys.IPO.Infrastructure.Tests/Repositories/InvitationRepositoryTests.cs

```diff
@@ -394,7 +394,8 @@ public void MoveCommPkg_ShouldChangeProjectRelationAndUpdateInvitation()
             Assert.AreNotEqual(toProjectName, GetProjectName(_mcPkg2.ProjectId));
 
             // Act
-            _dut.MoveCommPkg(_project3Guid, _commPkgGuid3Project1, description);
+            _dut.MoveCommPkg(_project3Guid, _commPkgGuid3Project1);
+            _dut.UpdateCommPkgDescriptionOnInvitations(_commPkgGuid3Project1, description);

```

Comment:
Strictly speaking you are testing two different concerns here, and they should have two separate unit tests.